src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+
+## What's defi?
+
+Defi is a collective term for products that are making financial products and services accessible to anyone who can use Ethereum – anyone with an internet connection. With defi, there are no centralized authorities who can block payments or deny you access to anything.
+
+There's a booming crypto economy out there, where you can lend, borrow, long/short, earn interest, and more. Crypto-savvy Argentinians have used defi to escape crippling inflation. Companies have started streaming their employees their wages in real time. Some folks have even taken out and paid off loans worth millions of dollars without the need for any personal identification.
+
+## Defi vs traditional finance
+
+One of the best ways to see the potential of defi is to understand the problems that exist today.
+
+- Some people aren't granted access to set up a bank account or use financial services
+- Financial services can block you from getting paid
+- The hidden charge of financial services is your personal data
+- Governments can close down markets at will
+- Money transfers can take days due to internal human processes
+- There's a premium to financial services because intermediray institutions need their cut
+
+| Defi                        | Traditional finance                                      |
+| --------------------------- | -------------------------------------------------------- |
+| You control your funds      | Your funds are looked after by companies                 |
+| Transfers happen in minutes | Payments can take days due to manual processes           |
+| Pseudonymous                | Financial activity is tightly coupled with your identity |
+| Open to anyone              | You must apply to use financial services                 |
+
+<ButtonLink to="/dapps/?category=finance">Explore defi apps</ButtonLink>
+
+## What can you do with defi?
+
+There's a decentralized alternative to most financial services. But Ethereum also creates opportunities for creating financial products that are completely new. This is an ever-growing list.
+
+- [Send money around the globe](#send-money)
+- [Stream money around the globe](#stream-money)
+- [Access stable currencies](#stablecoins)
+- [Lend/borrow funds](#lending)
+- [Borrow without collateral](#flash-loans)
+- [Start crypto savings](#savings)
+- [Swap tokens](#swaps)
+- [Trade??](#trading)
+- [Grow your portfolio](#investing)
+- [Fund your ideas](#crowdfunding)
+
+### Send money around the globe quickly {#send-money}
+
+As a blockchain, Ethereum is designed for sending transactions in a secure and global way. Payments go directly from one account to another in a matter of minutes (usually). To send/receive payments you'll need a wallet.
```

Comment:
I've always like the comparison of sending money anywhere in the world as easily as sending an email.